pom.xml

```diff
@@ -117,6 +117,17 @@ under the License.
       <artifactId>maven-shared-utils</artifactId>
       <version>3.4.2</version>
     </dependency>
+    <dependency>
+      <groupId>javax.inject</groupId>
+      <artifactId>javax.inject</artifactId>
```

Comment:
Shouldn't this be the following?

```xml
<dependency>
    <groupId>jakarta.inject</groupId>
    <artifactId>jakarta.inject-api</artifactId>
    <version>1.0.5</version>
</dependency>
```


https://mvnrepository.com/artifact/jakarta.inject/jakarta.inject-api

Reply:
No, that's a different package, fortunately